### PR TITLE
make csi-address can be configurable

### LINKED ISSUE #378: [FEATURES] Make csi-address can be configurable.
**What feature you'd like to add:**
Make csi-address can be configurable.
**Why is this feature needed:**
Because, admin maybe change kubelet default root-dir, the csi will can‘t register to kubelet.

### PR DESCRIPTION
**Pull request by wangandyang**
### Ⅰ. Describe what this PR does
make csi-address can be configurable

### Ⅱ. Does this pull request fix one issue?

fixes #378

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: charts/fluid/fluid/templates/csi/daemonset.yaml [hunk 1]
```diff
@@ -28,16 +28,16 @@ spec:
                 command: ["/bin/sh", "-c", "rm -rf /registration/fluid /registration/fuse.csi.fluid.io-reg.sock"]
           args:
             - --v=5
-            - --csi-address=/var/lib/kubelet/csi-plugins/fuse.csi.fluid.io/csi.sock
-            - --kubelet-registration-path=/var/lib/kubelet/csi-plugins/fuse.csi.fluid.io/csi.sock
+            - --csi-address={{ .Values.csi.kubelet.rootdir }}/csi-plugins/fuse.csi.fluid.io/csi.sock
```
Comment (xieydd) [inline charts/fluid/fluid/templates/csi/daemonset.yaml:31]: how about change rootdir to rootDir?

**Review by xieydd — CHANGES_REQUESTED** [review]

File: charts/fluid/fluid/templates/csi/daemonset.yaml (hunk 1 above)
Comment (wangandyang) [reply charts/fluid/fluid/templates/csi/daemonset.yaml]: it has changed

**Review by xieydd — APPROVED** [review]
/lgtm

**Comment by cheyang** [comment]
Thanks, @wangandyang

**Review by cheyang — APPROVED** [review]

**Pull request merged**